.github/workflows/release-bot.yml

```diff
@@ -0,0 +1,18 @@
+on: 
+  release:
+    types:
+      - created
+      
+name: Slack Notification on Release
+jobs:
+  slackNotification:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Slack Notification
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK}}
+        SLACK_COLOR: '#3278BD'
+        SLACK_MESSAGE: 'New helm charts for Hypertrace have been released :rocket:'
```

Comment:
Pl. add slack notification image as reference for notification

Reply:
Done